readme.md

```diff
@@ -77,7 +87,7 @@ With Fingerprint Pro Plus - you get access to five smart signals: Browser bot de
 
 ### Fingerprint Pro Use cases
 
-- [fingerprinthub.com](https://github.com/fingerprintjs/fingerprintjs-pro-use-cases) - Open-source demos of various Fingerprint Pro use cases including payment fraud prevention, promotion abuse prevention, credential stuffing protection, paywall metering, personalization, bot detection and more. 
+- [demo.fingerprint.com](https://github.com/fingerprintjs/fingerprintjs-pro-use-cases) - Open-source demos of various Fingerprint Pro use cases including payment fraud prevention, promotion abuse prevention, credential stuffing protection, paywall metering, personalization, bot detection and more. 
```

Comment:
I know this is how we had it before but isn't it a bit weird to link to a different page than the link in the text? Maybe we could have two links like:
[demo.fingerprint.com](https://demo.fingerprint.com) ([source](https://github.com/fingerprintjs/fingerprintjs-pro-use-cases)) - ....

Reply:
Good point, applied!